### ISSUE TITLE
yum error installink kmod-zfs on RHEL8

### ISSUE THREAD

**Issue by intentions** [zfsonlinux/zfs#9079]
### System information

Type | Version/Name
 --- | --- 
Distribution Name	|  Red Hat Enterprise Linux
Distribution Version	|  8.0
Linux Kernel	|  4.18.0-80.4.2.el8_0
Architecture	|  x86_64
ZFS Version	|  0.8.1-1
SPL Version	|  0.8.1-1

### Describe the problem you're observing

I'm trying to install zfs 0.8.1-1 on a RHEL 8 system.  I compiled the rpms from source but when I go to install the RPMs the system errors out installing the kmod rpms.  It says that it needs /sbin/depmod, but the system has /sbin/depmod, it also has a link in /usr/sbin/depmod to the same place

### yum log
 
```
Error: 
 Problem 1: conflicting requests
  - nothing provides /sbin/depmod needed by kmod-zfs-4.18.0-80.4.2.el8_0.x86_64-0.8.1-1.el8.x86_64
 Problem 2: package zfs-0.8.1-1.el8.x86_64 requires zfs-kmod = 0.8.1, but none of the providers can be installed
  - conflicting requests
  - nothing provides /sbin/depmod needed by kmod-zfs-4.18.0-80.4.2.el8_0.x86_64-0.8.1-1.el8.x86_64
 Problem 3: package zfs-test-0.8.1-1.el8.x86_64 requires zfs(x86-64) = 0.8.1-1.el8, but none of the providers can be installed
  - package zfs-0.8.1-1.el8.x86_64 requires zfs-kmod = 0.8.1, but none of the providers can be installed
  - conflicting requests
  - nothing provides /sbin/depmod needed by kmod-zfs-4.18.0-80.4.2.el8_0.x86_64-0.8.1-1.el8.x86_64
(try to add '--skip-broken' to skip uninstallable packages or '--nobest' to use not only best candidate packages)
```

**Comment by intentions** [comment]
I was able two work around this by making the changing lines 155 and 157 in zfs-0.8.1/scripts/kmodtool to the following:

Requires(post):   /usr/sbin/depmod
Requires(postun): /usr/sbin/depmod

**Comment by ofaaland** [comment]
Hi @intentions, this looks to me like a dupe of https://github.com/zfsonlinux/zfs/issues/8724 - if I'm incorrect, please point out the difference.  I'm looking at fix(es) for the RHEL 8 build/packaging issues.  Thanks!

**Comment by ofaaland** [comment]
Closing as duplicate of #8724 which was fixed by #9310
